ceph_deploy/mon.py

```diff
@@ -535,6 +528,26 @@ def make(parser):
 # Helpers
 #
 
+def get_mon_member(args, init_tags=False):
+    """
+    Abstracte the process about getting mon_member.
+
+    @init_tags for choose normal/init stage
+
+    return value are monitor members
+    """
```

Comment:
I really appreciate the documentation of input params and return values.  However, it's a bit inconsistent with the rest of the functions in ceph-deploy.  There is no where else that follows this style, especially with the @ symbol.

I'd rather just see a simple comment about what the function does, like:
```
Read and return the value of mon_initial_members from the Ceph config file

Optionally, a NeedHostError exception can be raised if the value is None.
```